src/main/java/co/omise/Example.java

```diff
@@ -492,6 +489,20 @@ void destroySchedule() throws ClientException, IOException, OmiseException {
         System.out.printf("destroyed schedule: %s", schedule.getId());
     }
 
+    void retrieveOccurrence() throws IOException, ClientException, OmiseException {
```

Comment:
How would you differentiate a normal occurrence from this?

Reply:
Exactly it same task. We can change that task to `wontfix`.

Reply:
Okk I'll get on that 👍